lxc/image.go

```diff
@@ -715,18 +715,10 @@ func (c *cmdImageImport) Run(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf(i18n.G("Only https:// is supported for remote image import"))
 	}
 
-	createArgs := &lxd.ImageCreateArgs{}
+	var createArgs *lxd.ImageCreateArgs
 	image := api.ImagesPost{}
 	image.Public = c.flagPublic
 
-	// Handle aliases
-	aliases := []api.ImageAlias{}
-	for _, entry := range c.flagAliases {
```

Comment:
Should aliases be handled here or is this a mistake?

Reply:
They only seem to matter in one case further down which had duplicated logic.